src/main/webapp/script.js

```diff
@@ -95,6 +95,32 @@ const gJSONusers = JSON.stringify(gUsers);
 const gDefaultUser = {userID: 0, name: 'Default Username',
   skills: 'Default Skills', major: 'Default Major',
   numTaskCompleted: 'Default Number'}; // Add attributes
+/**
+ * When the Project Page loads, get project info. If no projectID is provided in
+ * the URL, default values will be shown.
+ */
+function getProjectInfo() {
+  const params = new URLSearchParams(location.search);
+  const projectID = params.get('projectID');
+  const projects = JSON.parse(gJSONprojects);
+  for (project in projects) {
+    if (projects[project].projectID == projectID) {
+      const title = document.getElementById('project-title-container');
+      title.innerHTML = '<h1>' + projects[project].name + '</h1>';
```

Comment:
I didn't add this but when I pulled from master branch this line had an error and my whole javascript file was a bit messed up. Not sure what happened. Did I pull at the wrong time?